mindmaps-graph/src/test/java/io/mindmaps/graph/internal/MindmapsGraphHighLevelTest.java

```diff
@@ -258,19 +241,19 @@ public void getRelationTest(){
         roleMap.put(role1, rolePlayer1);
         roleMap.put(role2, rolePlayer2);
 
-        RelationImpl assertion = (RelationImpl) graph.addRelation(relationType).
+        RelationImpl assertion = (RelationImpl) mindmapsGraph.addRelation(relationType).
                 putRolePlayer(role1, rolePlayer1).putRolePlayer(role2, rolePlayer2);
-        Relation relationFound = graph.getRelation(relationType, roleMap);
+        Relation relationFound = mindmapsGraph.getRelation(relationType, roleMap);
         assertEquals(assertion, relationFound);
 
         assertion.getMappingCasting().forEach(casting -> {
-            Edge edge = graph.getTinkerPopGraph().traversal().V(casting.getBaseIdentifier()).inE(Schema.EdgeLabel.CASTING.getLabel()).next();
+            Edge edge = mindmapsGraph.getTinkerPopGraph().traversal().V(casting.getBaseIdentifier()).inE(Schema.EdgeLabel.CASTING.getLabel()).next();
```

Comment:
"PR for removing 'mindmaps' from the front of everything"
 
...proceeds to add 'mindmaps' to the front of everything